File: MeetingDoc.Api/Models/MeetingAgenda.cs
using System.Collections.Generic;

namespace MeetingDoc.Api.Models
{
    public class MeetingAgenda : BaseEntity
    {
        public int MeetingTimeId { get; set; }
        public MeetingTime MeetingTime { get; set; }
        public int Number { get; set; }
        public string Name { get; set; }
        public bool IsDraft { get; set; }
        public IList<MeetingAgendaUser> MeetingAgendaUsers { get; set; }
        public IList<MeetingContent> MeetingContents { get; set; }
    }
}

File: MeetingDoc-SPA/src/app/_models/MeetingAgenda.ts
import { BaseModel } from './BaseModel';
import { MeetingAgendaUser } from './MeetingAgendaUser';

export class MeetingAgenda extends BaseModel {
  meetingTimeId: number;
  number: number;
  name: string;
  users: MeetingAgendaUser[];
  isDraft: boolean;
}
